conjur/init/init_controller.py

```diff
@@ -45,24 +45,43 @@ def load(self):
         self.write_conjurrc()
 
         sys.stdout.write("Configuration initialized successfully!\n")
-        sys.stdout.write("To begin using the CLI, log in to the Conjur server by" \
-                         "running `conjur authn login`")
+        sys.stdout.write("To begin using the CLI, log in to the Conjur server by " \
+                         "running `conjur login`\n")
 
     def get_server_certificate(self):
         """
         Method to get the certificate from the Conjur endpoint detailed by the user
         """
         # pylint: disable=line-too-long
         if self.conjurrc_data.appliance_url is None:
-            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ")
+            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ").strip()
+            if self.conjurrc_data.appliance_url is None or self.conjurrc_data.appliance_url == '':
+                # pylint: disable=raise-missing-from
+                raise RuntimeError("Error: URL is required")
+
 
         url = urlparse(self.conjurrc_data.appliance_url)
+        # TODO: Factor out the following URL validation to ConjurrcData class
+        # At this time, providing ports is not supported.
+        # All HTTPS activity will be through 443
+        if url.netloc.find(":443") != -1 or (url.scheme != '' and url.path != ''):
```

Comment:
Why not use `.port`?
```suggestion
        if url.port == '' or (url.scheme != '' and url.path != ''):
```

Reply:
because https://something will also have a port = 443. Only when you attach that 443 to the url the netloc will have 443 which is why I have this condition